addons/website/static/src/js/content/menu.js

```diff
@@ -572,16 +575,17 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
 
         var alignment = this.defaultAlignment;
         if ($li.nextAll(':visible').length === 0) {
-            // The dropdown is the last menu item, open to the left
-            alignment = 'right';
+            // The dropdown is the last menu item, open to the left side
+            // (right side in rtl).
```

Comment:
I would say "with rtl languages" instead of "in rtl" (just nitpicking it can stay like this if you prefer :stuck_out_tongue: )